fitsio/test.py

```diff
@@ -219,6 +219,7 @@ def setUp(self):
                      {'name':'empty','value':''},
                      {'name':'test2','value':'stuff','comment':'this is a string keyword'},
                      {'name':'dbl', 'value':23.299843,'comment':"this is a double keyword"},
+                     {'name':'edbl', 'value':1.384123233e+43,'comment':"double keyword with exponent"},
```

Comment:
Do we want to add a '1.38E+03' test case here?

Reply:
for the zero in the 03?

Reply:
And the capital E

Reply:
well, cfitsio will write it how it writes it, putting a capital E in the python value won't change that.

BTW, it turns out the standard demands capital E in the header but cfitsio uses lower case, against the standard.

Reply:
ahhhh I thought these values went through the C code where you are testing for the capital E. I guess not. thanks!